<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>表格数据</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>当前记录</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSpinBox" name="dbSpinEmpNo"/>
          </item>
          <item>
           <widget class="QLineEdit" name="dbEditName"/>
          </item>
          <item>
           <widget class="QComboBox" name="dbComboSex"/>
          </item>
          <item>
           <widget class="QDateEdit" name="dbEditBirth"/>
          </item>
          <item>
           <widget class="QComboBox" name="dbComboProvince"/>
          </item>
          <item>
           <widget class="QComboBox" name="dbComboDep"/>
          </item>
          <item>
           <widget class="QSpinBox" name="dbSpinSalary"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="dbEditMemo"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_open"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="separator"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="separator"/>
   <addaction name="action_5"/>
   <addaction name="separator"/>
  </widget>
  <action name="act_open">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderOpen"/>
   </property>
   <property name="text">
    <string>打开数据库</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
   </property>
   <property name="text">
    <string>首记录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
   </property>
   <property name="text">
    <string>前一记录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
   </property>
   <property name="text">
    <string>后一记录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
   </property>
   <property name="text">
    <string>尾记录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
